src/api/stats.c

```diff
@@ -391,7 +391,7 @@ int api_stats_top_clients(struct ftl_conn *api)
 		// Return this client if either
 		// - "withzero" option is set, and/or
 		// - the client made at least one query within the most recent 24 hours
-		if(includezeroclients || count > 0)
+		if(includezeroclients || client_count > 0)
```

Comment:
This is the actual bugfix in here. We were using the wrong variable. `count` is the maximum number of clients to return